boot/zephyr/main.c

```diff
@@ -259,7 +259,7 @@ static void do_boot(struct boot_rsp *rsp)
 #endif
 
 #if defined(CONFIG_LOG) && !defined(CONFIG_LOG_IMMEDIATE) &&\
-    !defined(CONFIG_LOG_PROCESS_THREAD) && !defined(CONFIG_LOG_MINIMAL)
+    !defined(CONFIG_LOG_PROCESS_THREAD) && !defined(CONFIG_LOG_MODE_MINIMAL)
```

Comment:
CONFIG_LOG_MINIMAL is still emitted by zephyr, change here is not required, or maybe
```suggestion
    !defined(CONFIG_LOG_PROCESS_THREAD) && !defined(CONFIG_LOG_MINIMAL) && !defined(CONFIG_LOG_MODE_MINIMAL)
```